assembly/__tests__/ScaleString.spec.ts

```diff
@@ -41,9 +41,13 @@ describe("String", () => {
         const scaleString = ScaleString.fromU8a([0x04, 0x61]);
         expect<Array<u8>>(scaleString).toStrictEqual(([0x61]));
         expect<string>(scaleString.toString()).toStrictEqual("a");
+
+        const scaleString1 = ScaleString.fromU8a([0x08, 0x64, 0x61, 0x62, 0x62]);
+        expect<Array<u8>>(scaleString1).toStrictEqual([0x64, 0x61]);
+        expect<string>(scaleString1.toString()).toStrictEqual("da")
     });
 
-    itThrows("should throw on incorrect encoding", () => {
+    throws("should throw on incorrect encoding", () => {
```

Comment:
why did you change it to `throws`? 
We are using `itThrows` in many places, did the syntax of the `aspect` lib changed?

Reply:
Don't think it matters, so I'll just change it now